scrapi/base/__init__.py

```diff
@@ -40,6 +41,17 @@ def lint(self):
         return lint(self.harvest, self.normalize)
 
 
+class TransformerHarvester(BaseHarvester, Transformer):
```

Comment:
Not sure that I like this, seems unnecessary to define this class in addition to the transformer